python/tskit/provenance.py

```diff
@@ -27,6 +27,13 @@
 import json
 import os.path
 import platform
+import sys
+import time
+
+try:
+    import resource
+except ImportError:
```

Comment:
I think it's just that the module doesn't exist on Win right? Otherwise check wouldn't work.